README.md

```diff
@@ -1,60 +1,14 @@
-The course work will be cross-platform. However, in class the instructors will be working on Windows in both VSCode (for front end) and Visual Studio (for back end).
-- [nodejs & npm (LTS)](https://nodejs.org/en/download/) Node Package Manager will provide tooling for bulding our front end web site.
-
-### Recommended ###
-- [Visual Studio 2022](https://visualstudio.microsoft.com/downloads/). Version 17.5 or later. The community edition is fine, though we believe most students should have access to higher SKUs with EWU's MSDN (this assumption may be wrong). Lab computers should have Professional already installed. This will install .NET 7.0.
-- [GitKraken](https://gitkraken.keboo.dev/) Though you can do all of the git interaction from within Visual Studio or on the command line, GitKraken is free for open source work. It also provides a nice graphical version of the commit history so you can see how various commits and branches relate. 
-- [GitHub Student Developer Pack](https://education.github.com/students) Lots of great development tools and resources. The JetBrains products ReSharper (a plugin for Visual Studio), and Rider (a full C# IDE) are great tools for helping you write better code. 
-
-### VS Code Extensions used in class ###
-- [JEST](https://marketplace.visualstudio.com/items?itemName=Orta.vscode-jest) Unit testing extension
-- [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) code formatting extension
-- [Vetur](https://marketplace.visualstudio.com/items?itemName=octref.vetur) Vue tooling
-- [Code Spell Checker](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker)
+## Website link: ##
+[purple-rock-something-something.azurestuff.net](https://purple-rock-0cae9181e.3.azurestaticapps.net/)
+
+## Extra Credit Stuff - I think ##
+I'm not entirely 100% absolutely sure what the EC question is asking for but I'm gonna give it a good go.
+Some features to potentially be added:
+* An appearance/reskin system, so you can play the game with different styles of fonts/colorations/animations/etc (some loose theme ideas just off the top of my head: medieval manuscript, retro arcade, fuzzy friends, flower power, etc.)
+* Hints maybe? (Perhaps something like: sacrifice 1 guess for a letter that's in the word but no place is given or sacrifice 2 guesses for a letter who's place is given)
+* Hardmode/crazymode settings - less guesses, longer words, maybe after about 5-10 secs or so the letters of past guesses start to flutter around the page so if you don't remember what went where you'll just have to take a gamble.
+
+## Some Questions: ##
+* What is with the layout and why does it suck? (Seriously I can't figure out how to change this side-by-side nonsense and I hate it.)
+* Are you _really_ gonna mark me down 10pts for not working with somebody else?
+* Can we change the page title from "Vite App" or is it just... stuck that way?
```

Comment:
In your index.html you can change it by going to the

<title> "Your title here" <\title>

Reply:
In ordinary html file based pages yes, but I don't believe we have an `index.html` with this project. I might've just missed it, but to my knowledge the closest we had was `App.vue` which doesn't contain a < title > tag or even a proper < head > tag in which to place one.

Reply:
You actually do have one! It's in your main folder assignment1/index.html. It's just not in any subfolders.

Reply:
Oh, fascinating! I figured it would've/should've been in `src` folder with the other important files. Thanks for pointing that out!